### PR TITLE
Updated Okta document for IdP-initiated login notes

### PR DESCRIPTION
**Pull request by dahlsailrunner**
Hi guys -- I updated the Okta documentation to include a new step 5 to setup IdP-initiated logins.  That's the only change I made -- just the one Markdown file.  Let me know if this looks ok or if there is another way you'd like me to do this.

### CODE REVIEW

File: doc/IdentityServer3Okta.md [hunk 1]
```diff
@@ -102,4 +102,66 @@ public class Startup
...
 
 ## Step 4 - Try it out!  
-Logging in with Okta through your identity server should work at this point.  Cheers!
\ No newline at end of file
+Logging in with Okta through your identity server should work at this point.  Cheers!
+
+## Step 5 - Set up IdP-Initiated support 
+This is an optional but very nice and (I think) important step that will enable Okta users
+to login to your site by clicking on the "app" icon on their Okta Dashboard.  The process
+involves setting up the Kentor options to "allow unsolicited requests", and setting up a
```
Comment (AndersAbel) [inline doc/IdentityServer3Okta.md]: Should be "allow unsolicited response".

File: doc/IdentityServer3Okta.md [hunk 2]
```diff
@@ -102,4 +102,66 @@ public class Startup
...
+redirect page on your target site (not identity server), and specifying that in the Kentor
+settings.  Explanations follow.
+
+The first step is to configure the AuthServices/Acs endpoint to allow for unsolicited requests.
```
Comment (AndersAbel) [inline doc/IdentityServer3Okta.md]: reponse.

**Comment by AndersAbel** [comment]
Looks good, but I have one concern (except for the small typos) and that's the risk of a CSRF attack. What if a user is redirected to the ReturnUrl by a malicious site?

I think that the risk should be mentioned. It's hard to make a workaround without doing some major rewriting of AuthServices, so for now it can be there. In any case, the important CSRF protections is on sensitive operations in the client application.

Also, please don't submit pull requests from your master branch - make a separate branch. For this one it can be handled - but update your master branch from the upstream master until the PR is merged.

**Comment by dahlsailrunner** [comment]
I will refork and resubmit from a branch off of my forks' master.  I need the practice anyway.  :)
Regarding CSRF -- if someone went to my "IdP_InitiatedRedirect.aspx" page -- even with the idp=okta query string, I *think* we would be protected because that will be making an "authorize" request through both Identity server and then my custom user service.  If Okta (the IdP does not recognize a logged-in user, they will get stuck at a login screen.  If the user is not valid in my application (cannot look up roles, etc), then they won't get past my custom user service.  So I think we are safe.  I can add some notes to this effect in the doc if you like.
Let me know your thoughts and you can "reject" this PR and I will submit a new one.  Sorry for the trouble.

**Comment by AndersAbel** [comment]
You don't have to refork. Create a new branch in your repo from your current master (git checkout master; git checkout -b IdpInitiatedOkta). Then add KentorIT/AuthServices as another remote (called "upstream") to your local repo and reset your master to upstream/master (git remote add upstream https://github.com/KentorIT/AuthServices; git checkout master; git reset --hard upstream/master; git branch -u upstream/master) The final command will set your master to track the master in the upstream (i.e. master) repo. See also https://coding.abel.nu/2015/10/become-a-git-wizard-with-7-simple-tricks/ and https://coding.abel.nu/2015/03/fixing-a-pull-request-from-master/

I'm closing this PR as it is tied to your master branch. When you create a new branch, you will have to create a new PR.

**Pull request closed without merging**